pkg/internal/export/prom/prom.go

```diff
@@ -266,6 +284,30 @@ func newReporter(ctx context.Context, cfg *PrometheusConfig, ctxInfo *global.Con
 			Name: TracesTargetInfo,
 			Help: "target service information in trace span metric format",
 		}, labelNamesTargetInfo(ctxInfo)),
+		serviceGraphClient: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Name:                            ServiceGraphClient,
+			Help:                            "duration of client service calls, in seconds, in trace service graph metrics format",
+			Buckets:                         cfg.Buckets.DurationHistogram,
+			NativeHistogramBucketFactor:     defaultHistogramBucketFactor,
+			NativeHistogramMaxBucketNumber:  defaultHistogramMaxBucketNumber,
+			NativeHistogramMinResetDuration: defaultHistogramMinResetDuration,
+		}, labelNamesServiceGraph()),
+		serviceGraphServer: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Name:                            ServiceGraphServer,
+			Help:                            "duration of server service calls, in seconds, in trace service graph metrics format",
+			Buckets:                         cfg.Buckets.DurationHistogram,
+			NativeHistogramBucketFactor:     defaultHistogramBucketFactor,
+			NativeHistogramMaxBucketNumber:  defaultHistogramMaxBucketNumber,
+			NativeHistogramMinResetDuration: defaultHistogramMinResetDuration,
+		}, labelNamesServiceGraph()),
+		serviceGraphFailed: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Name: ServiceGraphFailed,
+			Help: "number of failed service calls in trace service graph metrics format",
+		}, labelNamesServiceGraph()),
+		serviceGraphTotal: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Name: ServiceGraphTotal,
+			Help: "number of service calls in trace service graph metrics format",
+		}, labelNamesServiceGraph()),
```

Comment:
Can we add a configuration option to enable/disable them? (I don't know which should be the default)

I'm thinking on users/customers not being users of App O11y that could see how their bill might be increased due to more metrics added unnoticedly.